modules/proc_manage-clairv4.adoc

```diff
@@ -0,0 +1,263 @@
+Linux operating systems. The micro services design of Clair v4 makes it
+appropriate to run in a highly scalable configuration, where
+components can be scaled separately as appropriate for enterprise environments.
+The Clair v4 link:https://github.com/quay/openshift-demo/blob/master/clairv4/config.yaml[config.yaml]
+illustrates how those services could run.
+
+For the purposes of trying out Clair v4, we recommend running it in combo mode (see
+link:https://github.com/quay/openshift-demo/blob/master/clairv4/clair-combo.yaml[clair-combo.yaml])
+This mode, described here, brings all the microservices together as one process.
+
+All supported security databases for Clair v4 are turned on. These databases include:
+
+* Alpine SecDB database
+* AWS UpdateInfo
+* Debian Oval database
+* Oracle Oval database
+* RHEL Oval database
+* SUSE Oval database
+* Ubuntu Oval database
+
+For information on how Clair does security mapping with the different databases, see
+link:https://quay.github.io/claircore/v0.0.19/severity_mapping.html[ClairCore Severity Mapping].
+
+[WARNING]
+====
+Because Clair v4 is Technology Preview, don't expect 100% accurate reporting.
+Expect that the presentation of vulnerability results will look different going
+forward than they did with v2. Over time, Clair v4 will produce more results.
+====
+
+The steps for using Clair v4 alongside an existing {productname} + Clair v2 environment are
+described next.
+
+== Configuring Clair v4
+
+To try out Clair v4, stand up a {productname} cluster
+with a running Clair v2 instance. Then use the following procedure
+to run Clair v4 along side it. Here is how to do that on an OpenShift v4.2 or later cluster.
+
+. Set your `$namespace` to the name of the project in which {productname} is running.
+For example:
++
+```
+$ export namespace=quay-enterprise
+```
+
+. Create a Postgres deployment file for Clair v4 (for example, `clairv4-postgres.yaml`)
+as follows.
++
+.clairv4-postgres.yaml
+[source,yaml]
+----
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: clairv4-postgres
+  labels:
+    quay-component: clairv4-postgres
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        quay-component: clairv4-postgres
+    spec:
+      volumes:
+        - name: postgres-data
+          persistentVolumeClaim:
+            claimName: clairv4-postgres
+      containers:
+        - name: postgres
+          image: postgres:latest
+          imagePullPolicy: "IfNotPresent"
+          ports:
+            - containerPort: 5432
+          env:
+            - name: POSTGRES_USER
+              value: "postgres"
+            - name: POSTGRES_DB
+              value: "clair"
+            - name: POSTGRES_PASSWORD
+              value: "postgres"
+          volumeMounts:
+            - name: postgres-data
+              mountPath: "/var/lib/postgres/data"
+---
+apiVersion: v1
+kind: PersistentVolumeClaim <1>
```

Comment:
remove `<1>`

Reply:
The <1> refers to the annotation under the yaml file.